tests/e2e/network/multiple_standard_lb.go

```diff
@@ -235,6 +238,36 @@ var _ = Describe("Ensure LoadBalancer", Label(utils.TestSuiteLabelMultiSLB), fun
 	})
 })
 
+func getDeploymentPodsNodeNames(kubeClient clientset.Interface, namespace, deploymentName string) (map[string]bool, error) {
```

Comment:
nit: add some comments for the new function

Reply:
done